tests/unit/specification/parser_test.py

```diff
@@ -2663,7 +2668,7 @@ def test_parse_error_invalid_range_aspect() -> None:
         end Test;
         """,
             r"^"
-            "<stdin>:2:42: parser: error: invalid Always_Valid expression: Invalid\n"
+            "<stdin>:2:26: parser: error: invalid Always_Valid expression: Invalid\n"
```

Comment:
Is it on purpose that the location points now to `Always_Valid` instead of `Invalid`?